### PR TITLE
Path planning

### PR DESCRIPTION
**Pull request by paulrosu11**

### CODE REVIEW

File: PathWeaver/output/Bean.wpilib.json [hunk 1]
```diff
@@ -0,0 +1,1847 @@
+[
```
Comment (akiyeng2) [inline PathWeaver/output/Bean.wpilib.json:1]: Do we want to check these into VC? I think probably just so if we transfer to a new computer we don't have to re-generate them

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -16,45 +36,74 @@
...
+            .andThen(new InstantCommand(() -> drivebase.tankDriveVolts(0, 0), drivebase));
+      } catch (IOException ex) {
+        DriverStation.reportError("Unable to open trajectory: " + pathName, ex.getStackTrace());
+      }
+      return null;
+    }
+
+    public Command getAutonomousCommand() {
+      return new SequentialCommandGroup(
+        generatePath("paths/Forward.wpilib.json"),
```
Comment (akiyeng2) [inline src/main/java/frc/robot/RobotContainer.java]: Won't this NullPointer if the path is not found? This error should be handled gracefully

File: src/main/java/frc/robot/commands/arcadeDrive.java [hunk 3]
```diff
@@ -38,7 +38,7 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    m_drivebaseSubsystem.arcadeDrive(speedValue.get() * (1 - throttleSupplier.get() * 0.3), rotaionValue.get() * (1 - throttleSupplier.get() * 0.3));
+    m_drivebaseSubsystem.arcadeDrive(speedValue.get(), rotaionValue.get());
```
Comment (akiyeng2) [inline src/main/java/frc/robot/commands/arcadeDrive.java]: spelling lol

File: src/main/java/frc/robot/subsystems/Drivebase.java [hunk 4]
```diff
@@ -57,6 +60,9 @@ public Drivebase() {
     m_leftEncoder = leftLeader.getEncoder();
     m_rightEncoder = rightLeader.getEncoder();
 
+    m_leftEncoder.setVelocityConversionFactor((1.0 / 60.0 / DrivebaseConstants.GEARING) * DrivebaseConstants.WHEEL_DIAMETER_METERS * Math.PI);
```
Comment (akiyeng2) [inline src/main/java/frc/robot/subsystems/Drivebase.java]: A brief explanation of where this is coming from would be nice, and what the final units are in

**Review by akiyeng2 — CHANGES_REQUESTED** [review]
Major thing that needs changing is the potential nullpointer if the path is not found. If that has been tested and isn't an issue, looks great!

File: src/main/java/frc/robot/commands/arcadeDrive.java (hunk 3 above)
Comment (ryan-huang1) [reply src/main/java/frc/robot/commands/arcadeDrive.java]: 😶‍🌫️

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (ryan-huang1) [reply src/main/java/frc/robot/RobotContainer.java:99]: there's a try, catch. Is there anything else we can do to make it better?

File: PathWeaver/output/Bean.wpilib.json (hunk 1 above)
Comment (ryan-huang1) [reply PathWeaver/output/Bean.wpilib.json:1]: VC as in main branch? I cleaned up a lot of the unused paths, however some still may be useful for testing.

**Review by fategg555 — APPROVED** [review]

**Review by ryan-huang1 — APPROVED** [review]

**Review by akiyeng2 — APPROVED** [review]

**Pull request merged**